src/widgets/basics/switch.h

```diff
@@ -31,7 +31,7 @@ class Switch : public QAbstractButton {
 
 protected:
     void paintEvent(QPaintEvent*) override;
-    void enterEvent(QEvent*) override;
+    void enterEvent(QEnterEvent*);
```

Comment:
Can you add the `override` back, so it is explicit ?
Ideally it should be added where its missing, but it's not a big deal if it's not...